tests/compare_utils.py

```diff
@@ -0,0 +1,41 @@
+# SPDX-FileCopyrightText: 2024 Pairinteraction Developers
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+from pathlib import Path
+from typing import TYPE_CHECKING, Optional
+
+import numpy as np
+
+if TYPE_CHECKING:
+    from pairinteraction.units import NDArray
+
+
+REFERENCE_PATHS = {
```

Comment:
Can't these paths be different for different tests? I would tend to specify them directly within the test_[...].py files.